packages/react-components/react-toolbar/library/src/components/ToolbarToggleButton/ToolbarToggleButton.types.ts

```diff
@@ -6,7 +6,7 @@ import { ToggleButtonProps, ButtonSlots, ToggleButtonState } from '@fluentui/rea
  */
 export type ToolbarToggleButtonProps = ComponentProps<ButtonSlots> &
   Partial<Pick<ToggleButtonProps, 'disabled' | 'disabledFocusable' | 'size'>> & {
-    appearance?: 'primary' | 'subtle';
+    appearance?: 'primary' | 'subtle' | 'transparent';
```

Comment:
please add VR tests for these new variants

Reply:
VR tests have been added